src/models/db/types.ts

```diff
@@ -6,7 +6,6 @@ export const tablesList = ['connection', 'connection_invite', 'query'] as const
 const insertConnection = z.object({
   company_name: z.string(),
   company_number: z.string(),
-  connection_id: z.string().optional(),
```

Comment:
I might have missed this, but is the field `connection_id` in table `connection` essentially unused now? An is it's functionality replaced by `agent_connection_id` in the `connection` table?

Reply:
it was never there in reality. The only reason this all worked was because it was optional and therefore always undefined